org.gnome.atomix.yaml

```diff
@@ -0,0 +1,32 @@
+app-id: org.gnome.atomix
```

Comment:
GNOME tends to capitalize the last part of the ID.

Reply:
See [1](https://gitlab.gnome.org/GNOME/atomix/-/blob/3.34.0/src/main.c#L690) and [2](https://gitlab.gnome.org/GNOME/atomix/-/commit/2b9b719582423251c14d0bfbfa2c81dc26e81b49#c9a4a258f31bf5583d4df244150a62a7e4a0aba0_0_2).

Reply:
and 3 https://github.com/flathub/flathub/pull/3146#discussion_r873214499